src/utils/translate/fr.json

```diff
@@ -453,6 +453,7 @@
     "Schulich": "École d'ingénieurs Schulich"
   },
   "ScientificLead": "Responsable scientifique",
+  "SelectDate": "Sélectionner une date...",
```

Comment:
Do we have this added in the German json as well?

Reply:
yup, just used Google Translate for now